src/main/resources/com/owlcyberdefense/mil-std-2045/xsd/milstd2045.enums.dfdl.xsd

```diff
@@ -0,0 +1,1067 @@
+      <xs:enumeration value="UNDEFINED_474" dfdlx:repValues="474"/>
+      <xs:enumeration value="UNDEFINED_475" dfdlx:repValues="475"/>
+      <xs:enumeration value="UNDEFINED_476" dfdlx:repValues="476"/>
+      <xs:enumeration value="UNDEFINED_477" dfdlx:repValues="477"/>
+      <xs:enumeration value="UNDEFINED_478" dfdlx:repValues="478"/>
+      <xs:enumeration value="UNDEFINED_479" dfdlx:repValues="479"/>
+      <xs:enumeration value="UNDEFINED_480" dfdlx:repValues="480"/>
+      <xs:enumeration value="UNDEFINED_481" dfdlx:repValues="481"/>
+      <xs:enumeration value="UNDEFINED_482" dfdlx:repValues="482"/>
+      <xs:enumeration value="UNDEFINED_483" dfdlx:repValues="483"/>
+      <xs:enumeration value="UNDEFINED_484" dfdlx:repValues="484"/>
+      <xs:enumeration value="UNDEFINED_485" dfdlx:repValues="485"/>
+      <xs:enumeration value="UNDEFINED_486" dfdlx:repValues="486"/>
+      <xs:enumeration value="UNDEFINED_487" dfdlx:repValues="487"/>
+      <xs:enumeration value="UNDEFINED_488" dfdlx:repValues="488"/>
+      <xs:enumeration value="UNDEFINED_489" dfdlx:repValues="489"/>
+      <xs:enumeration value="UNDEFINED_490" dfdlx:repValues="490"/>
+      <xs:enumeration value="UNDEFINED_491" dfdlx:repValues="491"/>
+      <xs:enumeration value="UNDEFINED_492" dfdlx:repValues="492"/>
+      <xs:enumeration value="UNDEFINED_493" dfdlx:repValues="493"/>
+      <xs:enumeration value="UNDEFINED_494" dfdlx:repValues="494"/>
+      <xs:enumeration value="UNDEFINED_495" dfdlx:repValues="495"/>
+      <xs:enumeration value="UNDEFINED_496" dfdlx:repValues="496"/>
+      <xs:enumeration value="UNDEFINED_497" dfdlx:repValues="497"/>
+      <xs:enumeration value="UNDEFINED_498" dfdlx:repValues="498"/>
+      <xs:enumeration value="UNDEFINED_499" dfdlx:repValues="499"/>
+      <xs:enumeration value="UNDEFINED_500" dfdlx:repValues="500"/>
+      <xs:enumeration value="UNDEFINED_501" dfdlx:repValues="501"/>
+      <xs:enumeration value="UNDEFINED_502" dfdlx:repValues="502"/>
+      <xs:enumeration value="UNDEFINED_503" dfdlx:repValues="503"/>
+      <xs:enumeration value="UNDEFINED_504" dfdlx:repValues="504"/>
+      <xs:enumeration value="UNDEFINED_505" dfdlx:repValues="505"/>
+      <xs:enumeration value="UNDEFINED_506" dfdlx:repValues="506"/>
+      <xs:enumeration value="UNDEFINED_507" dfdlx:repValues="507"/>
+      <xs:enumeration value="UNDEFINED_508" dfdlx:repValues="508"/>
+      <xs:enumeration value="UNDEFINED_509" dfdlx:repValues="509"/>
+      <xs:enumeration value="UNDEFINED_510" dfdlx:repValues="510"/>
+      <xs:enumeration value="UNDEFINED_511" dfdlx:repValues="511"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <simpleType name="retransmitIndicatorEnum" dfdlx:repType="tns:enum1">
+    <restriction base="tns:enumString">
+      <enumeration value="Original_Message" dfdlx:repValues="0"/>
+      <enumeration value="Retransmitted_Message" dfdlx:repValues="1"/>
+    </restriction>
+  </simpleType>
```

Comment:
In keeping with the way the other enums are ordered (per order of usage), should this be moved to between operationIndicatorEnum and messagePrecedenceEnum